sourcegraph/sourcegraph
# Add "CodeCopied" event log

Closes https://github.com/sourcegraph/sourcegraph/issues/37341.

## Description
This PR adds `CodeCopied` event log to analyze how often users copy code from results/files/notebooks.

## Test plan
- tested manually
- `sg start`
- Go to any file, search results, or notebook with code results/files
- Select some code and copy (either using the keyboard or right-click)
- Check-in DevTools.Networks tab that new `CodeCopied` event is sent
